### DISCUSSION TITLE
Java for Mac IOS 15

### DISCUSSION THREAD

**Discussion by stanleyloftness** [edwardkort/WWIDesigner#69] [Setup]
Java 15, which is the default version in the latest Mac IOS Sierra 10.12.6 will open WWI Designer, but nothing is functional

**Comment by edwardkort** [comment]
I don't know what "Java 15" is. The current version of Java is version 8 update 144. Go to java.com to install this version.

**Comment by edwardkort** [comment]
No confirmation of issue. Closing.

**Comment by NickAtNight62** [comment]
With the current MacOS (Monterey 12.7.6) and latest version of Java (Ve 8 Update 451 2025).  Software runs using:  java -jar WIDesigner-2.6.0.jar
